fixjsunit11/fix_js_unit11.js

```diff
@@ -0,0 +1,100 @@
+//bài 1: Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+function copyDataToArray1 (data, numLoop){
+  let result = [];
+    for(let i = 0; i < numLoop; i++){
+	  result.push(data);
+	}
+	return result;
+}
+function copyDataToArray2 (data, numLoop){
+  let result = [];
+    while(numLoop > 0){
+      result.push(data);
+	  numLoop--;
+	}
+	return result;
+}
+function copyDataToArray3 (data, numLoop){
```

Comment:
```suggestion
function copyDataToArray3 (data, numLoop) {
```
Số `3` ở đây nghĩa là gì?